mappings/net/minecraft/world/biome/layer/SetBaseBiomesLayer.mapping

```diff
@@ -16,9 +16,12 @@ CLASS clw net/minecraft/world/biome/layer/BiomeGroupToBiomeLayer
 	FIELD o TAIGA_ID I
 	FIELD p SNOWY_TAIGA_ID I
 	FIELD q OLD_GROUP_1 [I
-	FIELD r GROUP_1 [I
-	FIELD s GROUP_2 [I
-	FIELD t GROUP_3 [I
-	FIELD u GROUP_4 [I
+	FIELD r DRY_CLIMATE [I
+	FIELD s TEMPERATE_CLIMATE [I
+	FIELD t COOL_CLIMATE [I
+	FIELD u SNOWY_CLIMATE [I
```

Comment:
Not `_BIOMES`? It's an array of biomes *in* the climate. The "climate" would be the number.

Reply:
@Prospector suggested climate if I remember?

Reply:
I can change it if you want

Reply:
I think he meant it as a replacement for "group", not for "biomes".

Reply:
Changed it to _BIOMES